src/Service/Logger.service.js

```diff
@@ -201,6 +207,19 @@ export default class LoggerService extends DependencyAwareClass {
     this.logger.log('info', this.processMessage(message));
   }
 
+  /**
+   * Logs an error, using logger.error
+   * or logger.info based on the logging levels
+   * that are based on the process.env.DEPLOY_ENV
+   *
+   * @param error
+   */
+  warning(error) {
```

Comment:
`LoggerService.error` and `LoggerService.info` have different signatures. I'd rather go with the shared argument for this function rather than introduce ambiguity.